enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -208,20 +210,21 @@ func (c *ociContainer) Create(ctx context.Context, workDir string) error {
 	if err := c.createBundle(ctx, pid1); err != nil {
 		return status.UnavailableErrorf("create OCI bundle: %s", err)
 	}
-	// Create container
-	if err := c.invokeRuntimeSimple(ctx, "create", "--bundle="+c.bundlePath(), cid); err != nil {
+	// Create container (discard output as pid 1 is sleep, which inherits and never closes
+	// stdout/stderr)
```

Comment:
Hmm any idea why `create` is the thing that hangs, rather than `start`? I would think that the sleep process isn't created until we call `start`, but I'm not sure. At least if I do `podman create` it looks like that doesn't create the process

Reply:
I investigated this more and, if I'm not mistaken, `crun` starts the entrypoint command in `create`, not `start`. Since I also find this very surprising, I linked the call chain in a comment. I don't know whether `runc` or `podman` behave differently, we may need to extend the new handling to `start`.